assets/xml/objects/object_fu.xml

```diff
@@ -1,52 +1,61 @@
+        <Texture Name="gGuruGuruPantsTex" OutName="guru_guru_pants" Format="ci8" Width="16" Height="16" Offset="0x5060" TlutOffset="0x4C60"/>
+        <Texture Name="gGuruGuruHeadSkinTex" OutName="guru_guru_head_skin" Format="ci8" Width="8" Height="8" Offset="0x5160" TlutOffset="0x4E60"/>
+        <Texture Name="gGuruGuruMusicBoxTex" OutName="guru_guru_music_box" Format="ci8" Width="32" Height="32" Offset="0x51A0" TlutOffset="0x4C60"/>
+        <Texture Name="gGuruGuruFingersAndToesTex" OutName="guru_guru_fingers_and_toes" Format="ci8" Width="16" Height="16" Offset="0x55A0" TlutOffset="0x4E60"/>
+        <Texture Name="gGuruGuruEarTex" OutName="guru_guru_ear" Format="ci8" Width="8" Height="8" Offset="0x56A0" TlutOffset="0x4E60"/>
+        <Texture Name="gGuruGuruUpperHornAndCrankTex" OutName="guru_guru_upper_horn_and_crank" Format="ci8" Width="32" Height="32" Offset="0x56E0" TlutOffset="0x4C60"/>
+        <Texture Name="gGuruGuruShirtTex" OutName="guru_guru_shirt" Format="ci8" Width="32" Height="32" Offset="0x5AE0" TlutOffset="0x4C60"/>
+        <Texture Name="gGuruGuruHornAndPipeTex" OutName="guru_guru_horn_and_pipe" Format="ci8" Width="8" Height="8" Offset="0x5EE0" TlutOffset="0x4C60"/>
+        <Texture Name="gGuruGuruEyeClosedTex" OutName="guru_guru_eye_closed" Format="ci8" Width="32" Height="32" Offset="0x5F20" TlutOffset="0x4E60"/>
+        <Texture Name="gGuruGuruEyeAngryTex" OutName="guru_guru_eye_angry" Format="ci8" Width="32" Height="32" Offset="0x6320" TlutOffset="0x4E60"/>
+        <Texture Name="gGuruGuruMouthOpenTex" OutName="guru_guru_mouth_open" Format="ci8" Width="32" Height="16" Offset="0x6720" TlutOffset="0x4E60"/>
+        <Texture Name="gGuruGuruMouthAngryTex" OutName="guru_guru_mouth_angry" Format="ci8" Width="32" Height="16" Offset="0x6920" TlutOffset="0x4E60"/>
+        <Texture Name="gGuruGuruHairTex" OutName="guru_guru_hair" Format="ci8" Width="8" Height="16" Offset="0x6B20" TlutOffset="0x4E60"/>
+
+        <!-- Guru Guru Limbs -->
+        <Limb Name="gGuruGuruTorsoLimb" LimbType="Standard" EnumName="GURU_GURU_LIMB_TORSO" Offset="0x6BA0"/>
+        <Limb Name="gGuruGuruLeftThighLimb" LimbType="Standard" EnumName="GURU_GURU_LIMB_LEFT_THIGH" Offset="0x6BAC"/>
+        <Limb Name="gGuruGuruLeftLegLimb" LimbType="Standard" EnumName="GURU_GURU_LIMB_LEFT_LEG" Offset="0x6BB8"/>
+        <Limb Name="gGuruGuruLeftFootLimb" LimbType="Standard" EnumName="GURU_GURU_LIMB_LEFT_FOOT" Offset="0x6BC4"/>
+        <Limb Name="gGuruGuruRightThighLimb" LimbType="Standard" EnumName="GURU_GURU_LIMB_RIGHT_THIGH" Offset="0x6BD0"/>
+        <Limb Name="gGuruGuruRightLegLimb" LimbType="Standard" EnumName="GURU_GURU_LIMB_RIGHT_LEG" Offset="0x6BDC"/>
+        <Limb Name="gGuruGuruRightFootLimb" LimbType="Standard" EnumName="GURU_GURU_LIMB_RIGHT_FOOT" Offset="0x6BE8"/>
+        <Limb Name="gGuruGuruChestAndMusicBoxPipeLimb" LimbType="Standard" EnumName="GURU_GURU_LIMB_GHEST_AND_MUSIC_BOX_PIPE" Offset="0x6BF4"/>
```

Comment:
```suggestion
        <Limb Name="gGuruGuruChestAndMusicBoxPipeLimb" LimbType="Standard" EnumName="GURU_GURU_LIMB_CHEST_AND_MUSIC_BOX_PIPE" Offset="0x6BF4"/>
```

Reply:
Oh, the `G` -> `C`, it took me a bit to see it